components/InfoCard.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
```

Comment:
```suggestion
import React from 'react';
```
is there a reason to import everything vs the default module? [seems like it's the same for react](https://stackoverflow.com/questions/55285737/import-as-react-from-react-vs-import-react-from-react), just curious